stepflow-rs/tests/routing/README.md

```diff
@@ -0,0 +1,123 @@
+# Routing Tests
+
+This directory contains comprehensive tests and examples for the StepFlow routing system, demonstrating path transformation and component filtering capabilities.
+
+## Configuration Overview
+
+The `stepflow-config.yml` file demonstrates advanced routing features:
+
+### Complex Routing with Component Filtering
+```yaml
+- match: "/python/*"
+  target:
+    plugin: python
```

Comment:
This feels overly complicated. I'm trying to find a clean way to declare `/python/*` routes to the `python` plugin, and the component `/python/udf` corresponds to `/udf` within that plugin. The issue is how to describe that in a way that also allows printing the components, etc.